### PR TITLE
Update apiDefinition.swagger.json

### PR DESCRIPTION
**Pull request by jokano12**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2322,7 +2584,7 @@
           }
         ]
       }
-    },
+    ,
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: There is a missing closing curly bracket here

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 2]
```diff
@@ -2395,6 +2657,80 @@
           }
         ]
       }
+    },
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: There is an extra closing curly bracket here

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 3]
```diff
@@ -2395,6 +2657,80 @@
...
+              "$ref": "#/definitions/OrganizationData"
+            }
+          }
+        },
+        "summary": "List All Organizations",
+        "description": "List All Organizations",
+        "operationId": "List-All-Organizations",
+        "parameters": [
+          {
+            "name": "Page Number",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: The `name` parameter here should to be changed to `id`

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 4]
```diff
@@ -2395,6 +2657,80 @@
...
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "default": 1,
+            "description": "Page  Number.",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: There is an extra space in the description

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 5]
```diff
@@ -3781,6 +4117,1776 @@
     }
   },
   "definitions": {
+    "EmailData": {
+      "type": "object",
+      "description": "Email Data.",
+      "properties": {
+        "primary": {
+          "type": "boolean",
+          "description": "Primary Organization - Emails - Is Primary.",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: Please remove the `Primary Organization - Emails -` here as this is not a nested object.

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 6]
```diff
@@ -3781,6 +4117,1776 @@
...
+          "title": "Is Primary Email"
+        },
+        "address": {
+          "type": "string",
+          "description": "Primary Organization - Emails - Address.",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: Please remove prefix here too

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 7]
```diff
@@ -3781,6 +4117,1776 @@
...
+          "title": "Address"
+        },
+        "type": {
+          "type": "string",
+          "description": "Primary Organization - Emails - Type.",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: Please remove prefix here too

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 8]
```diff
@@ -3781,6 +4117,1776 @@
...
+          "title": "Type"
+        },
+        "showInDirectory": {
+          "type": "boolean",
+          "description": "Primary Organization - Emails - Show In Directory.",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: Please remove prefix here too

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 9]
```diff
@@ -80,6 +80,268 @@
...
+            "name": "Last Name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Last Name",
+            "x-ms-summary": "Last Name",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Page Number",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json:253]: For the Page Number, please add `"x-ms-visibility": "advanced"` and `"default": 1`

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 10]
```diff
@@ -80,6 +80,268 @@
...
+            "name": "ID Or Code",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Committee ID or Code",
+            "x-ms-summary": "Committee ID or Code",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Page Number",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json:312]: Also here
For the Page Number, please add `"x-ms-visibility": "advanced"` and `"default": 1`

File: certified-connectors/Impexium/apiDefinition.swagger.json (hunk 3 above)
Comment (lordlydon) [reply certified-connectors/Impexium/apiDefinition.swagger.json]: Also here
For the Page Number, please add `"x-ms-visibility": "advanced"` and `"default": 1`

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 11]
```diff
@@ -2395,6 +2657,80 @@
...
+            "type": "string",
+            "description": "Tag",
+            "x-ms-summary": "Tag"
+          },
+          {
+            "name": "Domain",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Organization Name.",
```
Comment (lordlydon) [inline certified-connectors/Impexium/apiDefinition.swagger.json]: Please update the description and summary

**Review by lordlydon — CHANGES_REQUESTED** [review]
Hi @jokano12,
I added a couple of comments, please review.

File: certified-connectors/Impexium/apiDefinition.swagger.json (hunk 3 above)
Comment (jokano12) [reply certified-connectors/Impexium/apiDefinition.swagger.json]: I removed the whole action "List all Organizations" as there are still internal decisions needed to be made.

Comment (lordlydon) [reply certified-connectors/Impexium/apiDefinition.swagger.json]: @RjImpx, is this going to be sent as part of a different PR, or should we wait until "List all Organizations" is ready to merge PR?

**Comment by RjImpx** [comment]
It can be part of a different PR so we can move forward.

[Visit Billhighway]
Raaj Jeeth
VP, Product Development
Phone: 877.486.8220
[https://cdn.billhighway.com/images/bh-gb+signature/facebook.png]<https://www.facebook.com/impexium>[https://cdn.billhighway.com/images/bh-gb+signature/twitter.png]<https://twitter.com/impexium>[https://cdn.billhighway.com/images/bh-gb+signature/linkedin.png]<https://www.linkedin.com/company/impexium>[https://cdn.billhighway.com/images/bh-gb+signature/rss.png]<https://www.impexium.com/>

**Review by lordlydon — APPROVED** [review]
Looks good

**Comment by jokano12** [comment]
Actions(5):
Get Committee Members by ID or Code
Find Members (Individuals) by First and Last Name
Add/Update address to Individual
Add Email to Individual
List All organizations

**Pull request merged**